src/test/java/bio/overture/ego/service/RefreshContextServiceTest.java

```diff
@@ -36,11 +37,34 @@
   @Rule public ExpectedException exceptionRule = ExpectedException.none();
 
   @Test
-  public void refresh_usedTokenIsDeleted() {
+  public void refresh_invalidContext_usedTokenIsDeleted() {
+    val user1 = entityGenerator.setupUserWithRefreshToken("User One");
+    // force invalid refreshContext with invalid user status
+    user1.setStatus(StatusType.PENDING);
```

Comment:
Our general guideline for static methods/fields is to statically import them. Intellij has an option that can do the static import for you